### PR TITLE
API phpunit 9 support

### PR DESCRIPTION
**Pull request by emteknetnz**
Issue https://github.com/silverstripe/silverstripe-framework/issues/10019

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,21 +1,21 @@
...
 
 matrix:
   fast_finish: true
   include:
-    - php: 5.6
+    - php: 7.3
       env: PHPUNIT_TEST=1 PHPCS_TEST=1
-    - php: 7.0
+    - php: 7.4
       env: PHPUNIT_TEST=1
```
Comment (maxime-rainville) [inline .travis.yml]:
```suggestion
    - php: 7.4
      env: PHPUNIT_TEST=1
    - php: 8.0
      env: PHPUNIT_TEST=1
```

If you want to go a step beyond, you could remove the `PHPUNIT_TEST` env var altogether and always run the build.

File: composer.json [hunk 2]
```diff
@@ -3,23 +3,18 @@
     "description": "Connects the PHP development server to SilverStripe",
     "license": "MIT",
     "require": {
-        "php": "^5.6||^7.0",
+        "php": "^7.3",
```
Comment (maxime-rainville) [inline composer.json]:
```suggestion
        "php": "^7.3 || ^8.0",
```

**Review by maxime-rainville — CHANGES_REQUESTED** [review]
Just need to update the module to officially support PHP8 and run a PHP8 build.

File: .travis.yml (hunk 1 above)
Comment (emteknetnz) [reply .travis.yml]: Updated

File: composer.json (hunk 2 above)
Comment (emteknetnz) [reply composer.json]: Updated

**Review by maxime-rainville — APPROVED** [review]
LGTM

**Pull request merged**